src/utils/logging_manager.py

```diff
@@ -60,6 +61,23 @@ def log(self, string, level: LoggingLevel = LoggingLevel.DEBUG):
         elif level == LoggingLevel.CRITICAL:
             self._LOG.critical(string)
 
+    def setEffectiveLevel(self, level: LoggingLevel):
+
+        # Note: pytest logging level cannot be higher than WARNING
+        # https://github.com/segevfiner/pytest/blob/master/
+        # _pytest/logging.py#L246
+
+        if level == LoggingLevel.DEBUG:
```

Comment:
Why do we need these if else conditions? 

Can we use : self._LOG.setLevel(level.value) instead?

Reply:
Good idea, done.